build.gradle

```diff
@@ -32,22 +41,25 @@ dependencies {
     implementation 'com.fasterxml.jackson.module:jackson-module-kotlin'
     implementation 'org.jetbrains.kotlin:kotlin-reflect'
     implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
-    implementation group: 'org.jetbrains.kotlinx', name: 'kotlinx-coroutines-core', version: "$coroutines_version"
-    implementation group: 'org.jetbrains.kotlinx', name: 'kotlinx-coroutines-reactive', version: "$coroutines_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-reactive:$coroutines_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-reactor:$coroutines_version"
     runtimeOnly 'org.springframework.boot:spring-boot-devtools'
 
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.+"
-    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit_version"
-    testImplementation "org.junit.jupiter:junit-jupiter-api:$junit_version"
     testImplementation "org.spekframework.spek2:spek-dsl-jvm:$spek_version"
     testRuntimeOnly "org.spekframework.spek2:spek-runner-junit5:$spek_version"
-    
+    testImplementation "io.projectreactor:reactor-test"
+    testImplementation "org.springframework.boot:spring-boot-starter-test"
+
     annotationProcessor "org.springframework.boot:spring-boot-configuration-processor"
 }
 
 test {
-    useJUnitPlatform {
-        includeEngines 'spek2'
+    useJUnitPlatform()
+
+    testLogging {
```

Comment:
My IntelliJ 2019.1 cannot run Integration test as it tries to use junit vintage for it, so I've added logging to see test statuses and run it with gradle